security/sanitizers.rst

```diff
@@ -0,0 +1,442 @@
+
+Adding sanitizers as part of the profile
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Another option is to add the sanitizer values as part of a profile. This way, you can easily switch between
+different configurations by using dedicated profiles.
+
+.. code-block:: ini
+   :caption: compiler_sanitizers/profiles/asan
+
+   include(default)
+
+   [settings]
+   build_type=Debug
+   compiler.sanitizer=Address
+
+   [conf]
+   tools.build:cflags+=["-fsanitize=address", "-fno-omit-frame-pointer"]
+   tools.build:cxxflags+=["-fsanitize=address", "-fno-omit-frame-pointer"]
+   tools.build:exelinkflags+=["-fsanitize=address"]
+   tools.build:sharedlinkflags+=["-fsanitize=address"]
+
+   [runenv]
+   ASAN_OPTIONS="halt_on_error=1:detect_leaks=1"
+
+For Visual Studio (MSVC) we can obtain an equivalent profile for AddressSanitizer:
+
+.. code-block:: ini
+   :caption: ~/.conan/profiles/asan
+
+   include(default)
+
+   [settings]
+   build_type=Debug
+   compiler.sanitizer=Address
+
+   [conf]
+   tools.build:cxxflags+=["/fsanitize=address", "/Zi"]
+   tools.build:exelinkflags+=["/fsanitize=address"]
+
+The Conan client is not capable of deducing the necessary flags from the settings and applying them automatically
+during the build process. It is necessary to pass the expected sanitizer flags according to the
+``compiler.sanitizer`` value as part of the compiler and linker flags.
+Conan's built-in toolchains (like ``CMakeToolchain`` and ``MesonToolchain``) will automatically
+pick up the flags defined in the ``[conf]`` section and apply them to the build.
+
+Managing sanitizers with a CMake toolchain
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Besides using Conan profiles to manage sanitizer settings, you can also use other approaches.
+
+If you already have a :ref:`custom CMake toolchain file <conan_cmake_user_toolchain>` to manage compiler
+and build options, you can pass the necessary flags to enable sanitizers there instead of profiles.
+
+.. code-block:: cmake
+   :caption: cmake/my_toolchain.cmake
+
+   # Apply to all targets; consider per-target options for finer control
+   set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fsanitize=address,undefined -fno-omit-frame-pointer")
+   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=address,undefined -fno-omit-frame-pointer")
+   set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -fsanitize=address,undefined")
+   set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -fsanitize=address,undefined")
+
+Then, specify this toolchain file as part of your Conan profile:
+
+.. code-block:: ini
+   :caption: profiles/asan_ubsan
+
+   include(default)
+
+   [settings]
+   build_type=Debug
+   compiler.sanitizer=AddressUndefinedBehavior
+
+   [conf]
+   tools.cmake.cmaketoolchain:user_toolchain=cmake/my_toolchain.cmake
+
+This way, you can keep your existing CMake toolchain file and still leverage Conan profiles to manage other settings.
+
+Building Examples Using Sanitizers
```

Comment:
I think all the example code would be better in the examples section, and linked from this page.